src/ontology/mondo-ingest.Makefile

```diff
@@ -430,6 +460,18 @@ $(COMPONENTSDIR)/%.db: $(COMPONENTSDIR)/%.owl
 	@rm -f .template.db.tmp
 	@rm -f $(COMPONENTSDIR)/$*-relation-graph.tsv.gz
 
+# TODO: Issue where Makefile $(COMPONENTSDIR)/% is overriding this goal
+#   This is here temporarily until issue fixed
```

Comment:
Why did this ever work in the past?

Reply:
[This comment](https://github.com/monarch-initiative/mondo-ingest/pull/299/files/dcb009428c6b1bc0f40620ab7247ea8d50f002aa#r1186924007) has the details. It's because there is a new `$(COMPONENTSDIR)/%` goal in `Makefile` that overrides `$(COMPONENTSDIR)/%-unmapped.owl`. However you have a solution where I will rename `unmapped/%-unmapped.owl`. Will implement shortly!

Reply:
`$(COMPONENTSDIR)/%-unmapped.owl` [changed to](https://github.com/monarch-initiative/mondo-ingest/pull/323) `unmapped/%-unmapped.owl`, fixing this issue for those artefacts.

However, `components/%.db` still won't run because of this recent ODK update, so I'm going to keep the `%-db` phony target as the way to run it. We can leave it like that for now, but ideally I'd either want to fix this if there's any way, or we can do what we did with `unmapped/` and make a different directory for our SemanticSQL component `.db`s.